release.py

```diff
@@ -4,8 +4,44 @@
 import ast
 import subprocess
 import sys
+from optparse import OptionParser
 
 DEBUG = False
+CONFIRM_STEPS = False
+DRY_RUN = False
+
+
+def skip_step():
+    """
+    Asks for user's response whether to run a step. Default is yes.
+    :return: boolean
+    """
+    global CONFIRM_STEPS
```

Comment:
@j-bennet I really like these two functions! Very handy.

The `global` statements aren't needed since `skip_step()` and `run_step()` are only referencing these two constants, not assigning to them.